src/Integration/LocalServices/ISourceControlledFileSystem.cs

```diff
@@ -35,11 +36,21 @@ internal interface ISourceControlledFileSystem : ILocalService
         /// <param name="fileWriteOperation">The actual write operation, return false if failed</param>
         void QueueFileWrite(string filePath, Func<bool> fileWriteOperation);
 
+        /// <summary>
+        /// Queues a write operation for given file paths. New or edit is determined internally at the time of this method execution.
```

Comment:
Not clear what "determined internally" means. Does it just means "evaluated"?

Reply:
Ah, it was a copy-paste of QueueFileWrite doc comment. I changed both to "evaluated"